requirements.txt

```diff
@@ -1,4 +1,12 @@
-flask
```

Comment:
So you are using `pip freeze` to create this? I don't think we need all items listed here and it seems that we missed the only necessary item, `flask-sqlalchemy`. It's better to _understand_ your dependency instead of generate it automatically.